### PR TITLE
Create allergy loop

### PR DESCRIPTION
**Pull request by annnash88**
@bontgoy - I'd like a code review please. :)

### CODE REVIEW

**Comment by Bontgoy** [comment]
Please wait to merge PR until you receive feedback if tagging me in a PR

File: ruby/vampires.rb [hunk 1]
```diff
@@ -39,6 +39,7 @@
 
   outcome = "Results inconclusive"
 
+
   if  ((age && garlic_bread && insurance) == true)
```
Comment (Bontgoy) [inline ruby/vampires.rb]: age is an undefined variable this will error

File: ruby/vampires.rb [hunk 2]
```diff
@@ -39,6 +39,7 @@
```
Comment (Bontgoy) [inline ruby/vampires.rb]: This condition should not be nested in the name check

File: ruby/vampires.rb [hunk 3]
```diff
@@ -55,10 +56,27 @@
     outcome = "Definitely a vampire"
   end
 
-  puts outcome
+
+  puts "What are your allergies?"
+  allergies = gets.chomp
```
Comment (Bontgoy) [inline ruby/vampires.rb]: you can only check allergies once as it is outside of the loop and ran again within the loop

File: ruby/vampires.rb [hunk 4]
```diff
@@ -55,10 +56,27 @@
...
+  until allergies == ("Done" || "sunshine") do # start of allergies loop
+    if  allergies == "Done"
+      puts "No allergies"
+    else
```
Comment (Bontgoy) [inline ruby/vampires.rb]: this sets allergies equal to sunshine if anything accept done is selected.

also the outcome is incorrect as this outcome is for the name check.

**Comment by Bontgoy** [comment]
@annnash88 Please review my comments and reach out at your soonest chance so that we can discuss how to get this control flow back on track

File: ruby/vampires.rb (hunk 1 above)
Comment (annnash88) [inline ruby/vampires.rb]: I thought I defined age on line 17. If that doesn't count, can you explain?

File: ruby/vampires.rb (hunk 3 above)
Comment (annnash88) [inline ruby/vampires.rb]: I only prompted the user once, but allergies continued to run until Done or sunshine was selected. In the future would I write until allergies == ( allergies = "Done" || allergies = "sunshine") do
puts "What are your allergies?"
if  allergies == "Done"
 +      puts "No allergies"
 +    else

File: ruby/vampires.rb (hunk 4 above)
Comment (annnash88) [inline ruby/vampires.rb]: What would I have wanted to put here to make this function?

I thought name outcome was on line 55?

**Pull request merged**